src/HealthChecks.NpgSql/NpgSqlHealthCheck.cs

```diff
@@ -37,7 +37,7 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             }
             catch (Exception ex)
             {
-                return new HealthCheckResult(context.Registration.FailureStatus, exception: ex);
+                return new HealthCheckResult(context.Registration.FailureStatus, description: ex.Message, exception: ex);

```

Comment:
I see such usage only in `ArrangoDbHealthCheck` but I'm fine to pass exception message into description arg.

Reply:
from our usage with a postgresql db, we get a null message in our log if healthcheck is unable to reach or connect to database. It ease the debug to know why it didn't succeed